code/datums/wires/vending.dm

```diff
@@ -9,7 +9,7 @@ var/const/VENDING_WIRE_IDSCAN = 8
 
 /datum/wires/vending/CanUse(var/mob/living/L)
 	var/obj/machinery/vending/V = holder
-	if(!istype(L, /mob/living/silicon))
+	if(!istype(L, /mob/living/silicon) && !L.lying && !L.incapacitated())
```

Comment:
Does this not mean I can just rest to not be shocked? You probably want to check user.stat for dead or dying people.